index.html

```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
             </div>
         </section>
         <!-- end of about -->
-        <!-- <section  class="get-in-touch">
-            <h1>Get in</h1>
-        </section> -->
+        <section  class="get-in-touch">
+            <h3 hidden>heading</h3>
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="/" method ="post">
+                <div class="input-control">
+                    <input class="fullname-control" type="text" id="full-name" placeholder="Full name" required>
```

Comment:
- [ ] As requested by the previous reviewer, Please add a `maxLength` attribute to your name field and set value to 30, So your name input field does not accept values more than 30 characters.